inventory_reporting/reports/inventory_report.py

```diff
@@ -87,48 +87,37 @@ def generate_valuation_report(self, workbook, wizard):
                 row += 1
 
     def generate_storable_report(self, workbook, wizard):
+        base_storable_domain = [
+            ["accounting_date", ">=", wizard.date_start],
+            ["accounting_date", "<=", wizard.date_end],
+            ["product_id.active", "=", True],
+            ["product_id.detailed_type", "=", "product"],
+        ]
```

Comment:
@AungKoKoLin1997 Is there a reason why this is done with lists (not with tuples)? If no specific reason, we should use tuples to go with the convention, and also to avoid unnecessary hiccups (not sure some methods like `expression.AND` works with domains with lists).